### PR TITLE
E2E Tests using Cypress

### PR DESCRIPTION
**Pull request by piedrabryan**
Implementing the 4 E2E Tests that were given in the Final Project Requirements. This includes code for the tests and the necessary libraries to run the tests.

### CODE REVIEW

File: cypress/e2e/spec.cy.js
Comment (BillquistErik) [inline cypress/e2e/spec.cy.js:1]: Not sure if keeping this file is necessary but probably won't cause any issues

File: cypress/e2e/chart.cy.js
Comment (BillquistErik) [inline cypress/e2e/chart.cy.js:1]: Looks good to me! well spaced and lots of notes

**Review by BillquistErik — COMMENTED** [review]
Nice Job!

**Review by jade-fleskes — COMMENTED** [review]
This looks good to me. There's a bunch of extraneous files in the commits, but I don't think that actually matters

File: cypress.config.js
Comment (CameronKroeker47) [inline cypress.config.js:1]: Thank you for remembering to correctly assign the baseUrl

File: cypress/e2e/chart.cy.js
Comment (CameronKroeker47) [inline cypress/e2e/chart.cy.js:1]: Lots of comments. Very clean.
Smart to use for loops. Makes it very compact and easy to read.

File: cypress/support/commands.js
Comment (CameronKroeker47) [inline cypress/support/commands.js:1]: Nice functions

**Review by CameronKroeker47 — APPROVED** [review]
Great job. The tests look great.

**Review by BillquistErik — APPROVED** [review]

**Review by BillquistErik — APPROVED** [review]

**Review by BillquistErik — APPROVED** [review]

**Pull request closed without merging**